packages/client/src/components/interface/Header/Header.tsx

```diff
@@ -527,12 +528,14 @@ class HeaderComp extends React.Component<IFullProps, IState> {
     )
 
     return (
-      <AppHeader
-        id="register_app_header"
-        desktopRightMenu={rightMenu}
-        title={title}
-        {...mobileHeaderActionProps}
-      />
+      <div {...headerProps}>
```

Comment:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/1206987/168548950-70e24062-18cf-4afe-ab26-6ab9db5dfbe0.png">

This breaks the top header's sticky positioning. Can we expand the props to AppHeader? Notice also that `this.props`has many non-html attributes that will end up rendered in the DOM, which causes console warnings. Can we either omit all non-html props from the object before expanding OR only explicitly take the ones we want to add to the element?

Reply:
I got you @rikukissa

Reply:
@rikukissa I was thinking to remove the DOM props Header and handle show/hide in RecordAudit using width state based on screen size.
`{width > grid.breakpoints.lg && <Header />}`
What do you think?

Reply:
Can you do it with CSS media queries? I think if yes, then that's the best approach 👍